include/deal.II/base/vectorization.h

```diff
@@ -134,12 +134,6 @@ class VectorizedArrayIterator
     return this->lane != other.lane;
   }
 
-  /**
-   * Copy assignment.
-   */
-  constexpr VectorizedArrayIterator<T> &
-  operator=(const VectorizedArrayIterator<T> &other) = default;
```

Comment:
Out of curiosity, why is it redundant? Because the compiler would generate this operator itself?

Reply:
Yes. If we rather want to be explicit we would define a defaulted copy constructor instead. I like following the rule of 0, though, if possible.